partner_delivery_zone/__manifest__.py

```diff
@@ -2,8 +2,9 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Partner Delivery Zone",
-    "summary": "Set on partners a zone for delivery goods",
-    "version": "14.0.1.1.0",
+    "summary": "This repo comprises modules that add functionalities"
```

Comment:
This description is not correct. This is not a repo, but a module, and the rest doesn't apply to the module goal